cmd/start.go

```diff
@@ -223,22 +224,28 @@ func init() {
 		&payoutPrivateKey,
 		"private-key",
 		"",
-		"[REQUIRED] Loadbalancers wallet private key, used for sending funds on payout",
+		"[REQUIRED] Load balancers wallet private key, used for sending funds on payout",
 	)
 
-	startCmd.Flags().Int32Var(
-		&payoutNumberOfDays,
-		"payout-interval",
-		0,
-		"[OPTIONAL] Payout interval in days, meaning each X days automatic payout will be executed")
-
 	startCmd.Flags().StringVar(
 		&payoutTotalReward,
 		"payout-reward",
 		"",
-		"[OPTIONAL] Total reward pool in Planck",
+		"[OPTIONAL] Total reward pool in Planck. If omitted, total balance of load balancer wallet will be considered as payout reward",
 	)
 
+	startCmd.Flags().StringVar(
+		&payoutFeeAddress,
+		"lb-payout-address",
+		"",
+		"[OPTIONAL] Address on which load balancer fee will be sent. If omitted, load balancer fee will be left on load balancer wallet after payout")
```

Comment:
I think we should make it required. If you leave fee on lb, it will be distributed in next payout?

Reply:
Or make it conditional so either payout-reward or lb-payout-address is required haha